src/components/App/App.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import App from './App';
+
+describe('App', () => {
+  let instance;
+  beforeEach(() => {
+    instance = render(<App />);
+  });
+
+  it('renders the title', () => {
+    expect(instance.getByText('Shopping List')).toBeInTheDocument();
+  });
+
+  it('renders the number of items on the list', () => {
+    expect(instance.getByText(/[0-9]+ items/)).toBeInTheDocument();
+  });
+
+  // it('renders the logo', () => {
+  //   expect(instance.getByTitle('Logo')).toBeInTheDocument();
+  // });
+
+  // it('renders the share icon', () => {
+  //   expect(instance.getByLabelText('Share this list')).toBeInTheDocument();
+  // });
```

Comment:
These tests wont pass until this PR is merged: https://github.com/mount-joy/thelist/pull/54

Reply:
#54 has been approved now! 😄